### ISSUE TITLE
Installation failed multible times

### ISSUE THREAD

**Issue by BenxProductionz** [kiwix/cardshop#247]
My Installation failed three times so far, it´s always failing with: 

> Installation failed: ssh command failed with status 2. cmd: sudo sh -c 'cd /var/lib/ansible/local && /usr/local/bin/ansible-playbook -vvv --inventory hosts --tags rename,reconfigure --extra-vars="@/var/lib/ansible/local/extra_vars.json" main.yml'
![fail](https://user-images.githubusercontent.com/47753465/115862890-fde79780-a434-11eb-958f-fd2cef2a92bd.JPG)

**Comment by kelson42** [comment]
@Popolechien  Any hint?

**Comment by Popolechien** [comment]
@kelson42 this looks like a desktop installer log. Might want to ping @rgaudin instead.

**Comment by BenxProductionz** [comment]
Yes, it´s the Windows desktop installer: https://download.kiwix.org/release/kiwix-desktop/kiwix-desktop_windows_x64.zip

**Comment by rgaudin** [comment]
Version ? it's in File -> About

**Comment by BenxProductionz** [comment]
It´s Version 2.3.8, now I use Version 2.0.7 and it works just fine.

**Comment by rgaudin** [comment]
OK, have you tried with latest one ? 2.4.0

**Comment by rgaudin** [comment]
btw your screenshot above is mostly useless to debug the issue as it doesn't inform about the actual problem. There's a *Copy log* button I think to share complete output. But don't bother if it's not on 2.4.0
